### ISSUE TITLE
itv.com: detection

### ISSUE THREAD

**Issue by 2Niko7B** [uBlockOrigin/uAssets#24530]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.itv.com/watch/love-island/2a3697`

### Category

detection

### Description

ITV and other streaming services have updated and are now detecting Ublock.

### Other extensions used

N/A

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.58.0
Chromium: 126
filterset (summary):
 network: 140284
 cosmetic: 47517
 scriptlet: 21758
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  easylist: 87085-49, now
  easyprivacy: 52810-657, now
  plowe-0: 3546-375, now
  ublock-badware: 9534-18, now
  ublock-filters: 39261-369, now
  ublock-privacy: 1037-5, now
  ublock-quick-fixes: 156-0, now
  ublock-unbreak: 2389-15, now
  urlhaus-1: 15380-0, now
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 631 ms (selfie)
 maxAssetCacheWait: 92 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 20
 network:
  amplitude.com: 19
  googletagmanager.com: 1
 extended:
  ##+js(json-prune, Playlist.ContentBreaks)
  ##+js(no-window-open-if)
```
</details>

**Comment by stephenhawk8054** [comment]
It's fixed already: https://github.com/uBlockOrigin/uAssets/commit/b405253ac7ba33101e546f59dff6367bf227d1dd

Update your filter lists again.

---

Also, please respect the prerequisites:

> - [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)

**Comment by 2Niko7B** [comment]
Fair play you are quick, promise I did look. Thanks so much for all you do <3Sent from my iPhoneOn 18 Jul 2024, at 12:16, stephenhawk8054 ***@***.***> wrote:﻿
It's fixed already: b405253
Update your filter lists again.

Also, please respect the prerequisites:


 I verified that this issue is not a duplicate. (Use this button to find out.)
